pkg/runtime/config.go

```diff
@@ -85,5 +85,8 @@ func (cfg *Config) Validate() error {
 	if cfg.AccountID == "" {
 		return errors.New("unable to start service controller as account ID is nil. Please pass --aws-account-id flag")
 	}
+	if cfg.Region == "" {
+		return errors.New("unable to start service controller as AWS region is nil. Please pass --aws-region flag")
+	}
```

Comment:
Initially, I added this for future work to support cross region and had to abandon it in lieu of CARM. That said, adding region argument does not change the current before. The SDK will still fetch region details from the environment variable.